SemiTruck.cpp

```diff
@@ -0,0 +1,17 @@
+
+#include "SemiTruck.h"
+#include "Vehicle.h"
+SemiTruck::SemiTruck(const std::string& n) : Vehicle(n)
+{
+    
+}
+
+void SemiTruck::setSpeed(int s)
+{
+    Vehicle::setSpeed(s);
+}
```

Comment:
If you are not going to customize that behavior, don't override base class member functions only to call the base class member function in your overridden function. 
- [ ] remove this function from `SemiTruck`